devtools-ci-index.yaml

```diff
@@ -2745,6 +2776,13 @@
             ci_cmd: 'TARGET="rhel" PR_CHECK_BUILD="true" /bin/bash ./.ci/cico_rhche_prcheck.sh'
             test_url: 'dev.rdu2c.fabric8.io'
             timeout: '60m'
+        - '{ci_project}-{git_repo}-rh-che-rollout-test-{test_url}':
```

Comment:
with that template name actual job would have confusing name 
`devtools-rh-che-rh-che-rollout-test-devtools-dev.ext.devshift.net` - 2 times `rh-che`

Reply:
I see, thanks, will fix that.